joser/film.html

```diff
@@ -86,12 +86,12 @@
                     <li>
                         <a href="http://mubi.com/films/the-shanghai-drama">The Shanghai Drama (1938)</a>
                     </li>
-                    <li>Le Monde En Armes (1939)
+                    <li><a href="http://www.imdb.com/title/tt0289741/?ref_=nm_flmg_dr_3">Le Monde En Armes (1939)</a>
                     </li>
                     <li>
                         <a href="http://mubi.com/films/salute-to-france">Salute To France (1944)</a>
                     </li>
-                    <li>Love Life of Adolph Hitler (1948)
+                    <li><a href="http://www.imdb.com/title/tt0204454/">Love Life of Adolph Hitler (1948)</a>
```

Comment:
This should be "Will It Happen Again?" that was the official release name.
The Canadian release was "The Strange Love Life of Adolf Hitler".

Both of these titles should be mentioned.